tests/requests/test_data_upload_settings.py

```diff
@@ -64,13 +64,13 @@ def test_no_limit(self):
             self.request._load_post_and_files()
 
     def test_file_passes(self):
-        payload = FakePayload("\r\n".join([
-            '--boundary',
-            'Content-Disposition: form-data; name="file1"; filename="test.file"',
-            '',
-            'value',
-            '--boundary--'
-            ''
+        payload = FakePayload(b"\r\n".join([
+            b'--boundary',
+            b'Content-Disposition: form-data; name="file1"; filename="test.file"',
+            b'',
+            b'value',
+            b'--boundary--',
+            b'',
```

Comment:
The original concatenated an empty string, but I believe the intention was to have two separate strings here so they can be joined by the `b"\r\n"`. If not, the empty string can go away. Hence the comma added to the `b'--boundary--'` line.